src/components/AvailableParkingSlotsView.tsx

```diff
@@ -13,12 +17,16 @@ interface IAvailableSlotsViewProps {
  * @returns A grid of Available Parking Slots
  */
 export const AvailableParkingSlotsView: React.FC<IAvailableSlotsViewProps> = ({
-  rows,
-  cols,
-  unavailableSlots,
+  parkingSlotsDB,
+  onConfigure,
 }) => {
+  const [selectedBuildingIndex, setSelectedBuildingIndex] = useState<number>(0);
+  const [selectedFloorIndex, setSelectedFloorIndex] = useState<number>(0);
```

Comment:
Could you please format your code below in this order - along with line spacing as well.
Component= () => {
    Hooks

    Function definitions

    Component logic
    
    return <></>
}

Reply:
Formatted the code such that it of the order as you mentioned and made sure line spacing look good as well